src/views/MyApps.vue

```diff
@@ -39,7 +39,10 @@
       class="no-auth-strategies-warning"
       data-testid="no-auth-strategies-warning"
     />
-    <AnalyticsConfigCheck require-analytics>
+    <AnalyticsConfigCheck
+      v-if="!myAppsReady || appsArePresent"
```

Comment:
`!myAppsReady` is the "not" intended here?

Reply:
Yes, the logic is:

- Apps are loading -> `myAppsReady` is `false` -> show the skeleton loader for the metric cards.
- If apps are present -> `appsArePresent` is `true` -> show the metric cards and let them issue the query.
- If apps are not present -> `appsArePresent` is `false` -> hide the metric cards.

I tested without `!myAppsReady` and the UX seemed jarring -- the metric cards would pop up as soon the table loaded and cause the table to shift downward.  It seems less jarring to hide them when the apps are not present.